tests/atest/rules/spacing/misaligned_continuation_row/expected_output.txt

```diff
@@ -1,7 +1,7 @@
-test.robot:5:19 [W] 1015 Each next continuation line should be aligned with the previous one
-test.robot:10:8 [W] 1015 Each next continuation line should be aligned with the previous one
-test.robot:19:19 [W] 1015 Each next continuation line should be aligned with the previous one
-test.robot:38:14 [W] 1015 Each next continuation line should be aligned with the previous one
-test.robot:41:10 [W] 1015 Each next continuation line should be aligned with the previous one
-test.robot:44:13 [W] 1015 Each next continuation line should be aligned with the previous one
-test.robot:58:14 [W] 1015 Each next continuation line should be aligned with the previous one
\ No newline at end of file
+test.robot:5:4 [W] 1015 Each next continuation line should be aligned with the previous one
```

Comment:
where it points now?

Reply:
Exactly where the issue occurs: 

<img width="708" alt="image" src="https://user-images.githubusercontent.com/7412964/226836429-6e3009f8-39d2-4c73-bfac-06be6e274191.png">